src/pages/ExamsPage/components/ExamSelection.test.jsx

```diff
@@ -43,9 +43,8 @@ describe('ExamSelection', () => {
     screen.getByText('Select an exam').click();
     const input = screen.getByPlaceholderText('Search for an exam...');
     await user.type(input, 'exam1');
-    expect(screen.getByText('exam1')).toBeInTheDocument();
-    expect(screen.queryByText('exam2')).not.toBeInTheDocument();
-    expect(screen.queryByText('exam3')).not.toBeInTheDocument();
+    const button = screen.queryByRole('link', { name: 'exam2' });
```

Comment:
I think we should check `exam1` is not disabled and `exam3` is disabled as well, for completeness.

Reply:
Good call! done.